job-engine/jbatch/src/main/java/org/eclipse/kapua/job/engine/jbatch/driver/JbatchDriver.java

```diff
@@ -264,6 +266,11 @@ public static boolean isRunningJob(@NotNull KapuaId scopeId, @NotNull KapuaId jo
         return getRunningJobExecution(scopeId, jobId) != null;
     }
 
+    public static void cleanJobData(@NotNull KapuaId scopeId, @NotNull KapuaId jobId) {
+        String jobName = getJbatchJobName(scopeId, jobId);
+        ((KapuaJDBCPersistenceManagerImpl)ServicesManagerImpl.getInstance().getPersistenceManagerService()).purgeByName(jobName);
```

Comment:
Wrap this in a `try catch` which catches `Exception` and throws `PurgeJobDataDriverException`